flask/wrappers.py

```diff
@@ -11,103 +11,12 @@
 
 from werkzeug.exceptions import BadRequest
 from werkzeug.wrappers import Request as RequestBase, Response as ResponseBase
+from werkzeug.wrappers.json import JSONMixin
```

Comment:
We'll still need to have a thin subclass around this, to adjust `json_module` to use `flask.json`.